code/modules/surgery/organs/organ_external.dm

```diff
@@ -680,6 +684,17 @@ Note that amputating the affected organ does in fact remove the infection from t
 		tbrute = 3
 	return "[tbrute][tburn]"
 
+/obj/item/organ/external/proc/update_splints() //we're processing splints like this instead of on a per step basis for efficiency
```

Comment:
"reacting" in this case would be making it do this every step, unless I'm missing something, which didn't seem efficient at all until I added the splinted_list thing, but even then I don't think the difference is noticable